### DISCUSSION TITLE
Crimson plank's Saturation

### DISCUSSION THREAD

**Discussion by KitaroHex** [BedrockTweaks/Files#364] [Ideas]
Well, to put it simple I miss the saturated crimson planks, making them more red, give more life to some builds, so Please add the saturated crimson planks

**Comment by SquatchHunter** [comment]
it used to be a pack on VT but they removed it for reasons, so to stay "truthful" to them we also removed it, however we have seen quite a bit of demand for it so we will probably be returning it soon
